recipe/bld.bat

```diff
@@ -1,22 +1,42 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat Docs Examples
+if errorlevel 1 exit 1
 
 cd ..
 if errorlevel 1 exit 1
 
 %PYTHON% "%RECIPE_DIR%\get_plugins.py"
 if errorlevel 1 exit 1
 
-copy "UAC\Plugins\x86-unicode\UAC.dll " "%PREFIX%\Plugins\x86-unicode\"
+copy "elevate\bin.x86-32\elevate.exe" "%PREFIX_NSIS%\Plugins\x86-unicode\"
```

Comment:
Do you know why this is installed in `Plugins`? Seems like it should go in `Bin` being an `exe` and all. 😕

Reply:
FWIW I know this is done in the Anaconda recipe. Just trying to understand the motivation for this choice.

Reply:
No idea, maybe @mingwandroid can enlighten us 😉